config/purify.php

```diff
@@ -4,132 +4,92 @@
-        |
-        | http://htmlpurifier.org/live/configdoc/plain.html#HTML.ForbiddenElements
-        |
-        */
-
-        'HTML.ForbiddenElements' => '',
-
-        /*
-        |--------------------------------------------------------------------------
-        | CSS.AllowedProperties
-        |--------------------------------------------------------------------------
-        |
-        | The Allowed CSS properties.
-        |
-        | http://htmlpurifier.org/live/configdoc/plain.html#CSS.AllowedProperties
-        |
-        */
-
-        'CSS.AllowedProperties' => 'font,font-size,font-weight,font-style,font-family,text-decoration,padding-left,color,background-color,text-align',
-
-        /*
-        |--------------------------------------------------------------------------
-        | AutoFormat.AutoParagraph
-        |--------------------------------------------------------------------------
-        |
-        | The Allowed CSS properties.
-        |
-        | This directive turns on auto-paragraphing, where double
-        | newlines are converted in to paragraphs whenever possible.
-        |
-        | http://htmlpurifier.org/live/configdoc/plain.html#AutoFormat.AutoParagraph
-        |
-        */
+    'configs' => [
+        'default' => [
+            'Core.Encoding' => 'utf-8',
+            'HTML.Doctype' => 'XHTML 1.0 Strict',
+            'HTML.Allowed' => 'h1,h2,h3,h4,h5,h6,b,strong,i,em,a[href|title],ul,ol,li,p[style],br,span,img[width|height|alt|src]',
+            'HTML.ForbiddenElements' => '',
+            'CSS.AllowedProperties' => 'font,font-size,font-weight,font-style,font-family,text-decoration,padding-left,color,background-color,text-align',
+            'AutoFormat.AutoParagraph' => false,
+            'AutoFormat.RemoveEmpty' => false,
+        ],
+    ],
 
-        'AutoFormat.AutoParagraph' => false,
+    /*
+    |--------------------------------------------------------------------------
+    | HTMLPurifier definitions
+    |--------------------------------------------------------------------------
+    |
+    | Here you may alter the HTML definitions used by HTMLPurifier. You can
+    | add elements and attributes as needed. We've already added a few
+    | common examples.
+    |
+    | Note that these definitions are applied to every Purifier instance.
+    |
+    | Documentation: http://htmlpurifier.org/docs/enduser-customize.html
+    |
+    */
 
-        /*
-        |--------------------------------------------------------------------------
-        | AutoFormat.RemoveEmpty
-        |--------------------------------------------------------------------------
-        |
-        | When enabled, HTML Purifier will attempt to remove empty
-        | elements that contribute no semantic information to the document.
-        |
-        | http://htmlpurifier.org/live/configdoc/plain.html#AutoFormat.RemoveEmpty
-        |
-        */
+    'definitions' => function(HTMLPurifier_HTMLDefinition $definition) {
```

Comment:
I think this will fail to cache or throw an exception when running `php artisan config:cache`?

Reply:
True, I'll convert is to a class reference

Reply:
Thanks!